schema/elasticsearch/visibility/versioned/v8/index_template_v7.json

```diff
@@ -0,0 +1,102 @@
+{
```

Comment:
Do not create the `v8` folder. This version corresponds to the Elasticsearch version, not the schema version.

Reply:
removed v8 folder